src/components/molecules/AppPageHeader.tsx

```diff
@@ -1,14 +1,13 @@
-import AppBackButton from "../atom/AppBackButton";
-import PageHeaderProps from "../../types/PageHeaderProps";
+interface PageHeaderProps {
+    title?: string | null;
```

Comment:
Why do we need to define a `null` value? title and description could be undefined is it really necessary to take into account `null` value? Also, does make sense title and description are optional? this component just prints the title and description I think at least one of them should be required

Reply:
@yodra The reason for the null is that usually these props are fed from a `translate(foo.bar)` call which retrieves the string according to the language being used, and the function can return a `null` value. To save a lot of typing in a lot of places I decided to allow the null value. What do you think?